fiware_cloto/environments/environment.py

```diff
@@ -103,15 +103,15 @@ def NotifyEmail(serverId, url, description, email):
         try:
             headers = {'Content-Type': 'application/json'}
             data = '{"action": "notifyEmail", "serverId": "' + serverId\
```

Comment:
Why don't you use python dicts and perform a _json.dumps_ when sending data into the HTTP POST request?